src/test/java/ca/team2706/frc/robot/commands/drivebase/DriverAssistVisionTest.java

```diff
@@ -75,42 +79,42 @@ public void setUp() throws NoSuchFieldException, IllegalAccessException {
      * is changed.
      */
     @Test
-    public void testGenerateTrajectoryCargoShipAndLoading() {
+    public void testGenerateTrajectoryCargoShipAndLoading(@Injectable("CARGO_AND_LOADING") DriverAssistVisionTarget target) {
```

Comment:
Not sure why we're mocking our own enum here.

Reply:
Also this variable isn't even getting used.

Reply:
We need a way to pass the type of target to the DriverAssist constructor. If there is an alternate way, feel free to change it.